.travis.yml

```diff
@@ -31,6 +31,13 @@ jobs:
         - make lint
         - make coverage
         - make validate-commit
+    - script:
+      - docker run -d --name bblfshd --privileged carlosms/bblfsh-ci-test:v0.0.1
+      - mkdir $HOME/repos
+      - git clone https://github.com/src-d/gitbase-playground.git $HOME/repos/gitbase-playground
```

Comment:
`gitbase` [relies](https://github.com/src-d/gitbase/blob/master/Gopkg.toml#L22T) in `gopkg.in/src-d/go-git-fixtures.v3` for testing; I think we could also do it

Reply:
Maybe I'm missing the point, but what would that provide to our tests? It looks to me like a storage for fixture files, querying that git repository from gitbase-playground will not add any edge cases or anything like that.

Reply:
These fixtures seem to be the company standards as they can be seen in `go-git` and `gitbase` itself.
I'd also say that these fixtures could be more stable than our own repo, whose history could be rewritten at any point.

Reply:
So you are not proposing to clone the https://github.com/src-d/go-git-fixtures repo to `$HOME/repos`, you want to use the fixture contents to initialize the directory that the gitbase docker container will serve? I think that's a bit too complex and should be considered in a separate PR

Reply:
We're testing against [the own history of this repo](https://github.com/src-d/gitbase-playground/pull/41/files#diff-88a7e4c84767081f5b182aef7e4f9f35L33); I'm proposing to test against a standard fixtures repo: `go-git-fixtures`

About the implication of that change... I'm not sure about its size; I'd say it would be mostly to change the fixtures repo, and to change the `blob.hash` being tested from `fd30cea52792da5ece9156eea4022bdd87565633` to a new one: `82f2e6eef212f7a349a2e79cfd46e5118ea64f95`

About doing it in this or other PR (if approved) it'd be up to you ;)

Reply:
As always, thanks for the suggestion. Since this PR is about adding the tests to Travis, I'd prefer to move this discussion to slack and merge this as soon as possible